src/plugins/gui/fiatPluginTypes.ts

```diff
@@ -101,6 +102,7 @@ export interface FiatPluginUi {
   addressForm: (params: FiatPluginAddressFormParams) => Promise<HomeAddress>
   sepaForm: (params: FiatPluginSepaFormParams) => Promise<SepaInfo>
   sepaTransferInfo: (params: FiatPluginSepaTransferParams) => Promise<void>
+  navigation: NavigationBase
```

Comment:
This ties the plugins pretty deeply to our app's internals. Is there some other way?